### ISSUE TITLE
EbaySectionNotice section not being announced by screen readers

### ISSUE THREAD

**Issue by pcanella** [eBay/ebayui-core-react#23]
Original issue: https://github.corp.ebay.com/React/ebayui-core-react/issues/397

**EbaySectionNotice** `section`  region (`role="region"`) is linking to a description empty element( `id="section-notice-attention"` element only contains a `<svg>`);

![image](https://user-images.githubusercontent.com/1072047/182663507-ac30960d-1e4a-44e5-9280-1d8e7141030f.png)

Following the solution reported in https://jirap.corp.ebay.com/browse/APOLLO-3398 and https://jirap.corp.ebay.com/browse/APOLLO-3395, the expected behavior is: "The region has an accessible name that it announced when the region is focused." - while currently the region is not announced by screen readers.

Issue created on ebayui-app-react: React/ebayui-app-react#152

**Comment by aaron-nance** [comment]
This isn't a bug. Just need to add the "aria-label" attribute to the tag.
